lib/features/home/presentation/widgets/recipe_list_tile.dart

```diff
@@ -40,7 +39,7 @@ class RecipeListTile extends StatelessWidget {
                 '${recipe.cookingDuration} min  -  ',
                 style: TextStyle(color: theme.colorScheme.onSurface),
               ),
-              DifficultyIndicator(difficulty: recipe.difficulty)
+              //DifficultyIndicator(difficulty: recipe.difficulty!)
```

Comment:
muss wieder rein :D (sollte nach merge conflicte lösen auch funktionieren.)
bitte ausrufezeichen nicht machen 
```suggestion
              //DifficultyIndicator(difficulty: recipe.difficulty ?? 1)
```